students/uss_tomasz/lesson_07_manipulating_strings_and_datetime_formatting/7_2/to_swap_the_two_words.py

```diff
@@ -0,0 +1,5 @@
+two_word_message = "Run Forrest"
+list_of_words = two_word_message.split()
```

Comment:
This is fine. Consider also following options

If you plan to use only first 2 elements of the list you can assign them specifically on split:
```
first_word, second_word = two_word_message.split()
```

Alternatively you could solve this with a one-liner that'd work for longer strings and won't raise an exception for empty or single word strings:
```
" ".join(reversed(two_word_message.split()))
```